tools-v2/README.md

```diff
@@ -981,6 +983,26 @@ Output:
 
 ```
 
+### update
+
+#### update peer
+
+reset peer
+
+Usage:
+```bash
+curve bs update peer 10.246.159.82:8200:0 --logicalpoolid=1 --copysetid=1
```

Comment:
It is best to hide the ip and port and replace it with 127.0.0.1:11234 or ***.***.***.***:****

Reply:
Yes, this is my negligence